viewing_party/party.py

```diff
@@ -1,23 +1,201 @@
 # ------------- WAVE 1 --------------------
 
 def create_movie(title, genre, rating):
-    pass
+    if title and genre and rating:
+        print("hello")
+        dict = {
+            "title": title,
+            "genre": genre,
+            "rating": rating
+        }
+        return dict
+
+    return None
+
+def add_to_watched(user_data, movie):
+    user_data_copy = user_data.copy()
+
+    # if movie:
+    user_data_copy["watched"].append(movie)
+
+    return user_data_copy

```

Comment:
The name of this function add_to_watched is explicitly telling us that it will **add** something so it's actually ok to directly modify `user_data["watched"]` in this scenario. It's also ok to append directly to `user_data["watched"]` since we're not in a for loop (we don't want to mutate an iterable when we're looping through it).

You can remove the comment on line 18 to clean up your code.